packages/web/src/components/GraphViewer.tsx

```
@@ -119,19 +157,33 @@ export default function GraphViewer({
       ref={containerRef}
-      className="rounded-lg border border-[var(--color-border)]"
+      className="relative rounded-xl border border-[var(--color-border)] bg-white shadow-sm overflow-hidden"
       style={{ height }}
-    />
+    >
+      <div className="absolute top-4 right-4 z-10 flex flex-col gap-2">
+         <button
+           className="rounded bg-white p-2 shadow-md hover:bg-slate-50 text-slate-600 border border-slate-100 transition-colors"
+           onClick={() => cyRef.current?.fit()}
+           title="Fit to screen"
+         >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)